### DISCUSSION TITLE
How do i show only the ai message

### DISCUSSION THREAD

**Discussion by Leo-Bon** [davidmigloz/langchain_dart#336] [Q&A] [answered]
res.choices.first.message 
I am using this to get answer to my prompt but it shows extra information like model:..., functioncall:... how do i achieve only the answer. Thanks in advance.

**Comment by davidmigloz** [comment, marked as answer]
Hello @Leo-Bon ,

I guess you are talking about `openai_dart` client?

If so, there's a `content` field in the `ChatCompletionAssistantMessage 

https://github.com/davidmigloz/langchain_dart/blob/c55fe50f0040cc04cbd2e90bca475887c093c654/packages/openai_dart/example/openai_dart_example.dart#L22-L36

**Reply by Leo-Bon** [reply]
Thanks a lot, I must have missed it on the documentary.
